examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,214 @@
+.. _examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain:
+
+Build a simple Autotools project using Conan
+============================================
+
+In this example, we are going to create a string formatter application
+that uses one of the most popular C++ libraries: `fmt <https://fmt.dev/latest/index.html/>`__.
+
+We'll use Autotools as build system and pkg-config as helper tool in this case, so you should get them installed
+on Linux and Mac before going forward with this example.
+
+Please, at first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/autotools/autotoolstoolchain/string_formatter
+
+We start from a very simple C++ language project with this structure:
+
+.. code-block:: text
+
+    .
+    ├── configure.ac
+    ├── Makefile.am
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic *configure.ac* including the **fmt** pkg-config dependency and the
+source code for the string formatter program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a default message, but uses ``fmt::print`` method for it.
+
+.. code-block:: cpp
+    :caption: **main.c**
+
+    #include <cstdlib>
+    #include <fmt/core.h>
+
+    int main(int argc, char** argv) {
+        fmt::print("{} - The C++ Package Manager!\n", "Conan");
+        return EXIT_SUCCESS;
+    }
+
+The ``configure.ac`` tells us about our project will a C++ compiler and a pkg-config module named as ``fmt.pc``
+
+.. code-block:: text
+    :caption: **configure.ac**
+
+    AC_INIT([stringformatter], [0.1.0])
+    AM_INIT_AUTOMAKE([1.10 -Wall no-define foreign])
+    AC_CONFIG_SRCDIR([src/main.cpp])
+    AC_CONFIG_FILES([Makefile])
+    PKG_CHECK_MODULES([fmt], [fmt])
+    AC_PROG_CXX
+    AC_OUTPUT
+
+And, the *Makefile.am* has ``string_formatter`` as our expected executable, and it will linked to the ``fmt`` library.
```

Comment:
```suggestion
And, the *Makefile.am* has ``string_formatter`` as our expected executable, and it will be linked to the ``fmt`` library.
```
passive voice?